mod_reforged/hooks/config/strings.nut

```diff
@@ -1290,6 +1291,20 @@ foreach (vanillaDesc in vanillaDescriptions)
 			]
 		}]
  	}),
+	RF_Inspire = ::UPD.getDescription({
+		Effects = [{
+			Type = ::UPD.EffectType.Passive,
+			Description = [
+				"Once per turn, after you use a skill targeting an adjacent ally, recover " + ::MSU.Text.colorGreen("2") + " [Action Points|Concept.ActionPoints]."
+			]
+		},
+		{
+			Type = ::UPD.EffectType.Active,
+			Description = [
+				"Gain the [Inspire|Skill+rf_inspire_skill] skill that triggers a positive morale check for an ally within 3 tiles with a bonus of 10% of your [Resolve|Concept.Bravery]. The same ally can only be inspired one time per turn."
```

Comment:
Because of nested tooltips, we don't need to give a full description of what the active skill does here. Just giving a general description of it is fine and the specifics can be found in the nested tooltip. I've been trying to do that in general with perk tooltips that give actives. For example you could write here `Gain the [Inspire|Skill+rf_inspire_skill] that allows you to target allies to increase their morale.` This way when we change the behavior of the inspire active skill we don't have to remember to modify the numbers in the string here as well.